test/unit/mock/objects.js

```diff
@@ -720,6 +720,66 @@ const installedStacksMock = {
   stack3: 'data3'
 }
 
+const auditLogItemMock = {
+  logs: {
+    uid: 'UID',
+    stack: 'blt8d542b122115b153',
+    created_at: 'created_at_date',
+    created_by: 'blt7b815b05d2fe5dd8',
```

Comment:
have you not installed **talismanrc** in your machine ? it avoid committing sensitive credentials